kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js

```diff
@@ -7,12 +7,12 @@ onEvent('recipes', (event) => {
         {
             output: 'industrialforegoing:fluid_collector',
             inputs: ['industrialforegoing:fluid_placer'],
-            id: 'industrialforegoing:fluid_collector'
```

Comment:
was this not overriding the default fluid collector recipe?

Reply:
this base recipe already was:
https://github.com/EnigmaticaModpacks/Enigmatica6/blob/01bcfb55a9dbe7290850702c22511429cf9e19cc/kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/industrialforegoing/shaped.js#L136

Reply:
This is a case of me forgetting to move the `base` recipe to `normal` scripts when it was changed for `expert`.

Reply:
Would you mind fixing that @Quezler? 😄

Reply:
not the entire file. Just that one recipe needs to be moved over to `kubejs/server_scripts/enigmatica/kubejs/normal/recipetypes/industrialforegoing/shaped.js`

Reply:
Fixed this in #4867. Just merge that into your stuff and you'll be set.

Reply:
Looks like you've merged that into here, i suppose it means its all good now?

Reply:
... Probably?